pmm-app/src/pmm-qan/panel/components/Filters/Filters.tsx

```diff
@@ -89,59 +93,44 @@ export const FiltersContainer: FC<FiltersContainerProps> = ({ filters, disabled,
   );
 
   return (
-    <div ref={filtersWrapperRef} className={cx({ [styles.filtersDisabled]: disabled })}>
-      <FiltersHeader />
-      <Scrollbar className={styles.getFiltersWrapper(height)}>
-        <FilterInput filter={filter} />
-        {FILTERS_GROUPS.filter((group) => filters[group.dataKey]).map(
-          ({ name, dataKey, getDashboardURL }) => (
-            <CheckboxGroup
-              key={name}
-              name={name}
-              items={filters[dataKey].name}
-              group={dataKey}
-              showAll={showAll}
-              filter={filter}
-              getDashboardURL={getDashboardURL}
-              rawTime={rawTime}
-            />
-          ),
-        )}
-      </Scrollbar>
-    </div>
-  );
-};
-
-export const Filters: FC = () => {
-  const {
-    contextActions,
-    panelState: { loadingDetails, rawTime },
-  } = useContext(QueryAnalyticsProvider);
-  const { filters, loading } = useFilters();
-  const initialValues = useInitialFilterValues();
-
-  return useMemo(
-    () => (
-      <Form
-        onSubmit={() => {}}
-        initialValues={initialValues}
-        render={({ form, handleSubmit }) => (
-          <Spin spinning={loading}>
-            <form
-              onSubmit={handleSubmit}
-              onChange={() => {
-                contextActions.setLabels(form.getState().values);
-              }}
-              onReset={() => {
-                contextActions.resetLabels();
-              }}
-            >
-              <FiltersContainer filters={filters} disabled={loadingDetails} rawTime={rawTime} />
-            </form>
-          </Spin>
-        )}
-      />
-    ),
-    [contextActions, filters, loading, loadingDetails, initialValues, rawTime],
+    <Form
+      onSubmit={() => {}}
```

Comment:
Can this onSubmit be removed? (I know it was already there bu since it's not doing anything)

Reply:
As far as I know, we must set it for the Final form, pls let me know if you know the way how to remove it